components/script/dom/workerglobalscope.rs

```diff
@@ -690,6 +692,7 @@ impl WorkerGlobalScope {
                     reports_chan.send(ProcessReports::new(reports));
                 });
             },
+            CommonScriptMsg::ReportCspViolations(..) => {},
```

Comment:
This seems surprising. What happens if a worker spawns another worker but that second worker is blocked by CSP? Does this line mean we silently drop that violation report?

Reply:
I thought that `CommonScriptMsg` could also be sent from the parent to the child worker. E.g. if a window creates a worker, then the window can also send `CommonScriptMsg::ReportCspViolations` to that worker. I didn't know any use case for it, hence I hadn't implemented this method yet. Is my understanding wrong for `CommonScriptMsg`? I will search for a WPT test that has nested workers to see if we can test this.